src/api/app/controllers/concerns/webui/requests_filter.rb

```diff
@@ -17,11 +17,13 @@ def filter_requests
     params[:creator] = @filter_creators if @filter_creators.present?
     params[:states] = @filter_state if @filter_state.present?
     params[:types] = @filter_action_type if @filter_action_type.present?
+    params[:stag_proj] = @filter_staging_project if @filter_staging_project.present?
```

Comment:
Why don't we use the full name? Instead of `stag_proj` -> `staging_project`, it's not that we're going to get charged more for the long variable name :sweat_smile: !